packages/side-nav/test/typings/side-nav.types.ts

```diff
@@ -0,0 +1,15 @@
+import '../../vaadin-side-nav.js';
+import '../../vaadin-side-nav-item.js';
```

Comment:
Currently the `vaadin-side-nav-item` is not present in the PR. So we can remove the import, or include the side nav item.